allennlp/models/vision_text_model.py

```diff
@@ -196,7 +193,7 @@ def forward(
         else:
             extended_attention_mask = None
 
-        extended_image_attention_mask = image_attention_mask.unsqueeze(1).unsqueeze(2).float().log()
+        extended_image_attention_mask = box_mask.unsqueeze(1).unsqueeze(2).float().log()
```

Comment:
You confirmed that this actually does something useful?

Reply:
The mask is necessary.